### ISSUE TITLE
update dependency versions

### ISSUE THREAD

**Issue by dportabella** [iipc/openwayback#367]
the pom.xml still has a dependency on hadoop-core 0.20.2-cdh3u4 from 2011, and com.sleepycat:je:4.1.6.
these versions are quite old, and are clashing with other dependency versions from my projet.

is it possible to update the dependency versions of openwayback pom.xml?
```
                       <dependency>
				<artifactId>hadoop-core</artifactId>
				<groupId>org.apache.hadoop</groupId>
				<version>0.20.2-cdh3u4</version>
			</dependency>

```

**Comment by ldko** [comment]
Hi @dportabella, It would be great to update the dependencies. Have you looked into whether upgrading these dependencies here would break anything/require changes beyond the pom.xml? Thanks, Lauren

**Comment by dportabella** [comment]
Yes, it does not compile anymore. There is some work to make it work when upgrading these dependencies.

**Comment by ldko** [comment]
I figured that would be the case considering how old the versions in use are, but wishful thinking... Are you interested in working on this issue? We would be happy to get a PR to fix this.

**Comment by dportabella** [comment]
too busy for the next six months :(

**Comment by ldko** [comment]
yeah, I know how that is... Thanks for reporting the issue and perhaps someone will be able to pick it up in the meantime. Any volunteers?
